templates/boundary/abstract.html

```diff
@@ -2,13 +2,24 @@
 {% block content %}
 <h3>Boundary data</h3>
 <p>These data are intended for exposing the polygon boundaries used across SNAP tools and data. Each query for a polygon will return the representative GeoJSON.</p>
-<h4>Service Endpoints</h4>
-<ul>
-    <li><a href="/boundary/climatedivision/">Climate Divisions</a>: Alaska Climate Divisions</li>
-    <li><a href="/boundary/corporation/">Corporations</a>: Alaska Native Corporations</li>
-    <li><a href="/boundary/ethnolinguistic/">Ethnolinguistic Regions</a>: Ethnolinguistic Regions of Alaska and parts of Canada.</li>
-    <li><a href="/boundary/firemanagement/">Fire Management</a>: Wildfire Management Zones for Alaska</li>
-    <li><a href="/boundary/huc/">Hydrologic Units (HUCs)</a>: HUCs of Alaska</li>
-    <li><a href="/boundary/protectedarea/">Protected Area</a>: Protected areas (parks, refuges, reserves, etc.) of Alaska, Yukon, and British Columbia.</li>
-</ul>
+<h4>Service Endpoint</h4>
+<p>The endpoint is <code>boundary/area/&ltvar_id&gt</code> for all <a href="/places/all">available area of interest ID codes</a>.</p>
```

Comment:
Use `&lt;` etc for HTML entities, I literally had no idea this would even work!  Maybe Jinja allows.  Whatever, I fixed in a commit.